### DISCUSSION TITLE
Momento de Sabedoria #7

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#433] [Imagens]
Post de referência; https://www.vish.blog.br/imagens/momento-de-sabedoria-7/

**Comment by brunoaugusto** [comment]
![null](<https://www.reactiongifs.us/wp-content/uploads/2013/04/popcorn_jon_stewart.gif>)

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/momento-de-sabedoria-7/#comment_1160593098>):

> [![null](<https://camo.githubusercontent.com/b3d9d391d086561fb97a33f0b1f0919f11c343f30dfbfd0dfb95ea41e4ac9952/68747470733a2f2f7777772e7265616374696f6e676966732e75732f77702d636f6e74656e742f75706c6f6164732f323031332f30342f706f70636f726e5f6a6f6e5f737465776172742e676966>)](<https://camo.githubusercontent.com/b3d9d391d086561fb97a33f0b1f0919f11c343f30dfbfd0dfb95ea41e4ac9952/68747470733a2f2f7777772e7265616374696f6e676966732e75732f77702d636f6e74656e742f75706c6f6164732f323031332f30342f706f70636f726e5f6a6f6e5f737465776172742e676966>)

Em minha defesa quem disse isso foi o Tom Jobim, vai lé reclamar com as caveiras dele :stuck\_out\_tongue:

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/momento-de-sabedoria-7/#comment_1160644932>):

> Em minha defesa quem disse isso foi o Tom Jobim, vai lé reclamar com as caveiras dele 😛

Você vai ver só... :triumph:



[![null](<https://camo.githubusercontent.com/62e2271d396fb36c7ae30f73590cc768a2a2233b66a7abd4003e2baa822ef9fb/68747470733a2f2f692e696d6775722e636f6d2f4b5065704747332e706e67>)](<https://camo.githubusercontent.com/62e2271d396fb36c7ae30f73590cc768a2a2233b66a7abd4003e2baa822ef9fb/68747470733a2f2f692e696d6775722e636f6d2f4b5065704747332e706e67>)

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/momento-de-sabedoria-7/#comment_1160720821>):

> Você vai ver só... 😆[![null](<https://camo.githubusercontent.com/62e2271d396fb36c7ae30f73590cc768a2a2233b66a7abd4003e2baa822ef9fb/68747470733a2f2f692e696d6775722e636f6d2f4b5065704747332e706e67>)](<https://camo.githubusercontent.com/62e2271d396fb36c7ae30f73590cc768a2a2233b66a7abd4003e2baa822ef9fb/68747470733a2f2f692e696d6775722e636f6d2f4b5065704747332e706e67>)

Foi sem querer querendo
